net/redistest/redistest.go

```diff
@@ -15,6 +15,9 @@ func NewTestRedis(t testing.TB) (address string, done func()) {
 }
 
 func NewTestRedisWithPassword(t testing.TB, password string) (address string, done func()) {
+
+	t.Skip("https://github.com/testcontainers/testcontainers-go/issues/1359")
```

Comment:
Can we set some deadline after which this skip won't work?
Something like this:
```
if time.Now() < time.Parse("2023-Sep-01", "2014-Feb-04") {
    t.Skip("https://github.com/testcontainers/testcontainers-go/issues/1359")
}
```

Reply:
Good idea, fixed.